app/assets/javascripts/angular-libs/ngs3upload.js

```diff
@@ -147,18 +147,23 @@ angular.module('ngS3upload.directives', []).
       require: '?ngModel',
       replace: true,
       transclude: false,
-      scope: true,
-      controller: ['$scope', '$element', '$attrs', '$transclude', function ($scope, $element, $attrs, $transclude) {
-        $scope.attempt = false;
-        $scope.success = false;
-        $scope.uploading = false;
+      scope: {
+        s3UploadPlacePic: '&'
+      },
+      controller: ['$scope', '$element', '$attrs', '$transclude',
+        function ($scope, $element, $attrs, $transclude) {
```

Comment:
'$transclude' is defined but never used.<br>'$attrs' is defined but never used.<br>'$element' is defined but never used.